lib/plugins/Plugins.json

```diff
@@ -1,5 +1,6 @@
 {
   "plugins": [
-    "./HelloWorld/HelloWorld.js"
+    "./HelloWorld/HelloWorld.js",
+    "./Tests/Tests.js"
```

Comment:
I don't think it should be loaded by default, but the test itself should simply load the Plugin so it isn't loaded when we release to customers